net/prosody/Makefile

```diff
@@ -56,18 +53,13 @@ define Build/Configure
 		--with-lua="$(STAGING_DIR_HOSTPKG)" \
 		--with-lua-include="$(STAGING_DIR)/usr/include" \
 		--with-lua-lib="$(STAGING_DIR)/usr/lib" \
-		--cflags="$(TARGET_CFLAGS)" \
+		--cflags="$(TARGET_CFLAGS) $(FPIC) -std=gnu99" \
```

Comment:
Here's where `CFLAGS` from `configure` is overridden.  The patch is not necessary.  The same happens to `LDFLAGS` below.